src/OmniSharp.Roslyn.CSharp/Services/Buffer/ChangeBufferService.cs

```diff
@@ -0,0 +1,26 @@
+using System.Composition;
+﻿using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Buffer
+{
+    [OmniSharpHandler(typeof(RequestHandler<ChangeBufferRequest, object>), LanguageNames.CSharp)]
```

Comment:
It _may_ be possible, but it would outside how Mef seems to do things typically.  In theory we could take the string, check it against the list of end points, and then reflect over the type to ensure that it implements the correct interface.
It _may_ be possible, but it would outside how Mef seems to do things typically.  In theory we could take the string, check it against the list of end points, and then reflect over the type to ensure that it implements the correct interface.